iothub/device/src/DeviceClient.cs

```diff
@@ -37,22 +37,24 @@ private DeviceClient(InternalClient internalClient)
         /// </summary>
         /// <param name="hostname">The fully-qualified DNS hostname of IoT Hub</param>
         /// <param name="authenticationMethod">The authentication method that is used</param>
+        /// <param name="options">The options that allow configuration of the device client instance during initialization.</param>
         /// <returns>A disposable DeviceClient instance</returns>
-        public static DeviceClient Create(string hostname, IAuthenticationMethod authenticationMethod)
+        public static DeviceClient Create(string hostname, IAuthenticationMethod authenticationMethod, DeviceClientOptions options = default)
```

Comment:
You need to overload this. It is a breaking change otherwise.

Reply:
`DeviceClientOptions` is added as an optional parameter; it will not be a breaking change.